src/view/drag-handle/drag-handle.jsx

```diff
@@ -165,7 +168,7 @@ export default class DragHandle extends Component<Props> {
     }
 
     // this will check if anything else in the system is dragging
-    if (!this.canLift()) {
+    if (!this.canLift(this.props.draggableId)) {
```

Comment:
This is the fix. Previously this allowed drags to start while a drop was occurring on the same element (which is not supported).